packages/e2e-tests/package.json

```diff
@@ -1,5 +1,6 @@
 {
 	"name": "@wordpress/e2e-tests",
+	"private": true,
```

Comment:
Ideally, we should include a note in the package README, CHANGELOG file and publish them to npm to inform that the package is getting into the maintenance mode and won't get any more updates.

Reply:
A note was added in the README but not in the CHANGELOG.

Could we just [deprecate](https://docs.npmjs.com/cli/v8/commands/npm-deprecate) the package in the npm settings? Publishing them once more just to deprecate them again is so tedious 😅 .

Reply:
There is no mention on npm https://www.npmjs.com/package/@wordpress/e2e-tests because the change in the README file wasn't published. I still see some downloads for the package:

<img width="463" alt="Screenshot 2022-03-24 at 12 45 42" src="https://user-images.githubusercontent.com/699132/159909693-263e4de8-0591-4a26-b251-2971b431ba3e.png">

It would really be better to do it in steps so we don't disrupt other projects.